src/Pos/WorkMode.hs

```diff
@@ -17,6 +17,8 @@ module Pos.WorkMode
 import           Universum
 
 import           Control.Monad.Fix
+import           Control.Monad.Base             (MonadBase)
+import           Control.Monad.Trans.Control
```

Comment:
Alright, I didn't know it existed. Do you list and modify them by hand or use some sort of automation (afair GHC can generate explicit import lists, but I don't remember exactly how)? If it's the former, that sounds quite inefficient.